test/api_spec.js

```diff
@@ -1008,17 +1013,16 @@ describe("api", function () {
       return uploadImage({
         public_id: PUBLIC_ID_BACKUP_1,
         backup: true,
-      }).then(() => uploadImage({
+      }).then(wait(1000)).then(() => uploadImage({
```

Comment:
Not sure I get why running on a fresh cloud impacts the time we need to wait until we do something. 
Can you explain?

Reply:
No definitive answer I'm afraid.

It's either these tests were never 100% stable and this timeout increases their stability,
or maybe the fresh cloud still runs some background things that gives a slight delay to assets being uploaded.

Reply:
Don't like voodoos.
I suggest checking with someone from the BE to see if your assumption makes sense

Reply:
This was confirmed by the backend team, while they don't know for a fact what can cause the increased latency for a new cloud, we definitely feel it.